build.gradle

```diff
@@ -10,7 +10,7 @@ plugins {
 }
 
 group 'uk.gov.hmcts.reform'
-version '4.3.1'
+version '4.4.1'
```

Comment:
4.4.0?

Reply:
new feature rather than a patch fix ?

Reply:
it's a new feature so its .4 but you should reset the patch number to 0